src/tests/testClpMap.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_BASE_TESTCLPMAP_H
+#define SQUID_BASE_TESTCLPMAP_H
+
+#include "compat/cppunit.h"
+#include "base/ClpMap.h"
```

Comment:
sort please. `b` before `c`
```suggestion
#include "base/ClpMap.h"
#include "compat/cppunit.h"
```

Reply:
done